tests/ttnn/unit_tests/gtests/tensor/test_vector_conversion.cpp

```diff
@@ -115,5 +130,68 @@ TEST(FloatVectorConversionTest, RoundtripBfloat16Representation) {
     }
 }
 
+class BlockFloatVectorConversionTest : public ::testing::TestWithParam<DataType> {};
+
+TEST_P(BlockFloatVectorConversionTest, InvalidLayout) {
+    ttnn::SimpleShape shape{128, 128};
+    EXPECT_ANY_THROW(
+        Tensor::from_vector(std::vector<float>(shape.volume()), get_tensor_spec(shape, GetParam(), Layout::ROW_MAJOR)));
```

Comment:
please leave a comment, this throws because block float and row major layout are not compatible